src/main/java/uk/gov/hmcts/divorce/common/model/State.java

```diff
@@ -93,7 +93,10 @@
         name = "Application issued",
         label = "# **${[CASE_REFERENCE]}** ${applicant1LastName} **&** ${applicant2LastName}\n### **${[STATE]}**\n"
     )
-    Issued("Issued");
+    Issued("Issued"),
+
+    ConditionalOrderComplete("ConditionalOrderComplete"),
+    FinalOrderComplete("FinalOrderComplete");
```

Comment:
there are no such states in divorce 1 so just thinking should we do it in another way ? @linusnorton would like to know your thoughts on this :)